package.json

```diff
@@ -128,6 +128,7 @@
     "run-time-error": "1.4.0",
     "secp256k1": "4.0.2",
     "shebang-loader": "0.0.1",
+    "sort-package-json": "^1.53.1",
```

Comment:
@Leeyoungone Please lock down the version to an exact one instead of a range so that we can have (or get closer to) [reproducible builds](https://reproducible-builds.org/)

Reply:
oo got it ! thanks :)